> please run `make pylint` locally and confirm it working, as travis is not currently running, and we intend to fix it later...

sure!

```
vatsa@vatsa-VirtualBox:~/kadalu/cli$ sudo make pylint 
cd kubectl_kadalu && python3 -m pylint --disable W0511,R0801 *.py
No config file found, using default configuration
************* Module install
C: 71, 0: Exactly one space required around assignment
            file_url="https://github.com/kadalu/kadalu/releases/download/%s" % args.version
                    ^ (bad-whitespace)
W: 75, 0: Bad indentation. Found 16 spaces, expected 12 (bad-indentation)
W: 67, 8: Unused variable 'version' (unused-variable)
************* Module storage_list
E: 11, 0: Bad option value 'useless-object-inheritance' (bad-option-value)
************* Module utils
E: 11, 0: Bad option value 'useless-object-inheritance' (bad-option-value)

------------------------------------------------------------------
Your code has been rated at 9.74/10 (previous run: 9.74/10, +0.00)

Makefile:17: recipe for target 'pylint' failed
make: *** [pylint] Error 22
vatsa@vatsa-VirtualBox:~/kadalu/cli$ pylint3 kubectl_kadalu/storage_list.py 
No config file found, using default configuration
************* Module storage_list
E: 11, 0: Bad option value 'useless-object-inheritance' (bad-option-value)

------------------------------------------------------------------
Your code has been rated at 9.67/10 (previous run: 9.67/10, +0.00)

vatsa@vatsa-VirtualBox:~/kadalu/cli$ python3 kubectl_kadalu/storage_list.py 
```
I get below error everytime...
```
vatsa@vatsa-VirtualBox:~/kadalu$ sudo make pylint
[sudo] password for vatsa: 
No config file found, using default configuration
************* Module kadalulib
E: 49, 0: invalid syntax (<string>, line 49) (syntax-error)
Makefile:61: recipe for target 'pylint' failed
make: *** [pylint] Error 2
```
The code is compiling and working properly.

### When pvc is present
```
vatsa@vatsa-VirtualBox:~/kadalu$ sudo kubectl get pvc
NAME   STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS      AGE
pv1    Bound    pvc-5686b9d2-70e9-4a33-9696-6c1a42603bcf   100Mi      RWO            kadalu.replica1   9s

vatsa@vatsa-VirtualBox:~/kadalu/cli$ sudo make release 
  adding: __main__.py (deflated 66%)
  adding: install.py (deflated 63%)
  adding: storage_add.py (deflated 71%)
  adding: storage_list.py (deflated 73%)
  adding: storage_remove.py (deflated 61%)
  adding: storage_yaml.py (deflated 67%)
  adding: utils.py (deflated 62%)
  adding: version.py (stored 0%)
Single deployment file is ready: build/kubectl-kadalu
vatsa@vatsa-VirtualBox:~/kadalu/cli$ cd build/
vatsa@vatsa-VirtualBox:~/kadalu/cli/build$ sudo python3 kubectl-kadalu storage-list --status 

Name             Type        Utilization            Pvs Count      Min PV Size      Avg PV Size      Max PV Size
storage-pool-1   Replica1    100 Mi/3 Gi (2%)               1           100 Mi           100 Mi           100 Mi  
```
### When pvc is not present
```
vatsa@vatsa-VirtualBox:~/kadalu$ sudo kubectl delete pvc/pv1
persistentvolumeclaim "pv1" deleted

vatsa@vatsa-VirtualBox:~/kadalu$ sudo kubectl get pvc
No resources found in default namespace.

vatsa@vatsa-VirtualBox:~/kadalu/cli/build$ sudo python3 kubectl-kadalu storage-list --status 

Name             Type        Utilization            Pvs Count      Min PV Size      Avg PV Size      Max PV Size
storage-pool-1   Replica1    0/3 Gi (0%)                    0                0                0                0  
``` 
All values are set to default without raising ValueError.

--- a/cli/kubectl_kadalu/storage_list.py
+++ b/cli/kubectl_kadalu/storage_list.py
@@ -207,10 +207,13 @@ def fetch_status(storages, args):
             # num_pvs|used_size|min_pv_size|avg_pv_size|max_pv_size
             pv_stats_parts = parts[1].strip().split("|")
             storage.pv_count = int(pv_stats_parts[0])
-            storage.used_size_bytes = float(pv_stats_parts[1])
-            storage.min_pv_size = float(pv_stats_parts[2])
-            storage.avg_pv_size = float(pv_stats_parts[3])
-            storage.max_pv_size = float(pv_stats_parts[4])
+
+            if storage.pv_count:
+                storage.used_size_bytes = float(pv_stats_parts[1])
+                storage.min_pv_size = float(pv_stats_parts[2])
+                storage.avg_pv_size = float(pv_stats_parts[3])
+                storage.max_pv_size = float(pv_stats_parts[4])
+
         except utils.CommandError as err:
             print("Failed to get size details of the "
                   "storage \"%s\"" % storage.storage_name,

